### PR TITLE
RSS plugin!

### PR DESCRIPTION
**Pull request by rockneurotiko**
RSS plugin for #35 and all of us :-)
There are one luarocks dependency (feedparser) and two system dependencies (for feedparser).
Included in launch.sh and README.md :smile: 

I've been using it for a week and works perfectly.

### CODE REVIEW

**Comment by carapapa** [comment]
Hi, I had a proble because of version < 5.3 in lua. Need to add local feedparser = require "feedparser".
My system is RPi2 with lua 5.2.

How can we set the time interval in rss asking?

Thank you for that great work!

**Comment by rockneurotiko** [comment]
Hi @carapapa 
I'm testing it in a new virtual machine to have a clean install, it's an ubuntu 12.04.
Following the README, all was installed correctly, I have lua 5.2.0, and feedparser installed correctly, this is the installation log:
(Remember to install expat libexpat1-dev in the system)
```
Installing https://rocks.moonscript.org/feedparser-0.71-3.src.rock...
Using https://rocks.moonscript.org/feedparser-0.71-3.src.rock... switching to 'build' mode

Missing dependencies for feedparser:
luaexpat

Using https://rocks.moonscript.org/luaexpat-1.3.0-1.src.rock... switching to 'build' mode
gcc -O2 -fPIC -I/usr/include/lua5.2 -c src/lxplib.c -o src/lxplib.o -I/usr/include -Isrc/
gcc -shared -o lxp.so -L/usr/lib src/lxplib.o -L/usr/lib -Wl,-rpath,/usr/lib: -lexpat
Updating manifest for /home/vagrant/telegram-bot/.luarocks/lib/luarocks/rocks
luaexpat 1.3.0-1 is now built and installed in /home/vagrant/telegram-bot/.luarocks (license: MIT/X11)

Updating manifest for /home/vagrant/telegram-bot/.luarocks/lib/luarocks/rocks
feedparser 0.71-3 is now built and installed in /home/vagrant/telegram-bot/.luarocks (license: BSD)
```

The require yeah, I'm doing it now, thanks for the remember :)

And the interval.. Right now is 5 minutes, the cron timer, but if it's a good feature I can rethink the way the plugin adds to the postpone function (I already have an idea) to have a custom time :)

File: README.md [hunk 1]
```diff
@@ -295,7 +300,7 @@ Installation
 ------------
 ```bash
 # Tested on Ubuntu 14.04, for other OSs check out https://github.com/vysheng/tg#installation
-sudo apt-get install libreadline-dev libconfig-dev libssl-dev lua5.2 liblua5.2-dev libevent-dev make unzip git redis-server g++ libjansson-dev libpython-dev
+sudo apt-get install libreadline-dev libconfig-dev libssl-dev lua5.2 liblua5.2-dev libevent-dev make unzip git redis-server g++ libjansson-dev libpython-dev expat libexpat1-dev
```
Comment (yagop) [inline README.md]: Where is required `expat` and `libexpat1-dev`?
I installed and tested `feedparser` through `luarocks` and its working

**Comment by carapapa** [comment]
Thank you @rockneurotiko

I found the postpone time set in bot.lua, I have set up for 2 minutes. Other plugins that use cron function (isup.lua, i.e.) use that ok, but rss.lua in my system is not following that time setup, I don't know why. It is like a random time calling. Where can I find a rss.lua working log?
I have no response with "!rss sync" command neither. 
My system is rapsbian (Raspberry Pi 2).

Thank you!

File: README.md (hunk 1 above)
Comment (rockneurotiko) [inline README.md]: When I installed feedparser from luarocks in my raspberry, in the dependency of feedparser, that is `luaexpat`, it failed to install, and searching they said that I needed that two libraries, so I installed and then feedparser intalled ok from luarocks.
Maybe you already have then installed, or they are no longer needed

**Comment by rockneurotiko** [comment]
@carapapa that's strange, in my system is working ok :S 

You can see if it's calling every 5 minutes if you do a print, or write some log to a file with the time, and see if it's working.

Also, the plugin use redis to store the configuration, if you don't have it, maybe you can see some errors :S 

To force the sync and see if everything is working, I usually go inside redis and change the value that know what the last one is.

For example:
```
$ redis-cli
127.0.0.1:6379> KEYS rss:*:last_entry
1) "rss:www.commitstrip.com/en/feed/:last_entry"
2) "rss:finofilipino.org/rss:last_entry"
3) "rss:bartoszmilewski.com/feed/:last_entry"
4) "rss:www.haskellforall.com/feeds/posts/default?alt=rss:last_entry"
127.0.0.1:6379> SET rss:finofilipino.org/rss:last_entry ""
OK
```
Then I just do !rss sync and sync send me the last ten:
captura de pantalla de 2015-05-31 17-42-00

**Comment by Brawl345** [comment]
What do I have to change to get the post itself and post an entry in one message? e.g. I have two posts and I want the bot to post two messages?

**Comment by rockneurotiko** [comment]
@Brawl345 Changing the code, the cron function. Actually, look after this line: https://github.com/rockneurotiko/telegram-bot/blob/plugins/rss/plugins/rss.lua#L136

**Comment by carapapa** [comment]
@rockneurotiko Your plugin is running perfectly. Maybe the rss source was the failure in the update frecuency. I have been checking from yesterday with others rss and everything is ok. I changed the time in bot.lua and it works flawlessly. Thank you!

**Comment by rockneurotiko** [comment]
Good news @carapapa ! :smile: 
I think that maybe, when subscribe or in the cron, if the plugin don't recognise the structure of the RSS (should be standard, but it's not always true), won't work well.

**Pull request merged**